source/Octopus.Tentacle/Maintenance/WorkspaceCleaner.cs

```diff
@@ -52,18 +52,18 @@ public async Task Clean(CancellationToken cancellationToken)
                     {
                         continue;
                     }
+                    
+                    //First look to see if there is a log file and if we can delete the workspace based on that
```

Comment:
Potentially the changes in https://github.com/OctopusDeploy/OctopusTentacle/pull/919 make this code clearer.

Reply:
yeah, I think I'll pull your change in

Reply:
cherry picked into this PR